apps/elasticsearch/liveness/test.yml

```diff
@@ -0,0 +1,99 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - vars.yml
+
+  tasks:
+    - block:
+        - block:
+
+            - name: Record test instance/run ID
+              set_fact:
+                run_id: "{{ lookup('env','RUN_ID') }}"
+
+            - name: Construct testname appended with runID
+              set_fact:
+                test_name: "{{ test_name }}-{{ run_id }}"
+
+          when: lookup('env','RUN_ID')
+        
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - include_tasks: /utils/fcm/update_litmus_result_resource.yml
+          vars:
+            status: 'SOT'
+
+        - name: Getting the application pod name
+          shell: kubectl get po -n {{ namespace }} -l {{ app_label }} -o jsonpath={.items[0].metadata.name}
+          register: pod_name
+
+        - name: Replacing the placeholder for pod-name
+          replace:
+            path: "{{ elasticsearch_liveness }}"
+            regexp: "pod-name"
+            replace: "{{ pod_name.stdout }}"   
+
+        - name: Replacing the placeholder for namespace
+          replace:
+            path: "{{ elasticsearch_liveness }}"
+            regexp: "app-namespace"
+            replace: "{{ namespace }}"   
+
+        - name: Replacing the placeholder for liveness-retry-count
+          replace:
+            path: "{{ elasticsearch_liveness }}"
+            regexp: "liveness-retry-count"
+            replace: "{{ liveness_retry }}"   
+
+        - name: Replacing the placeholder for liveness-timeout
+          replace:
+            path: "{{ elasticsearch_liveness }}"
+            regexp: "liveness-timeout-seconds"
+            replace: "{{ liveness_timeout }}"  
+
+        - name : Fetch app service name
+          shell: kubectl get svc -n {{ namespace }} -o jsonpath={.items[0].metadata.name}
+          register: name_of_service 
+
+        - name: Getting the port 
+          shell: kubectl get endpoints -n {{ namespace }} -o jsonpath={.items[0].subsets[0].ports[0].port}
+          register: port_no
+
+        - name: Replace the placeholder for service endpoint
+          replace:
+            path: "{{ elasticsearch_liveness }}"
+            regexp: "service_name"
+            replace: "{{ name_of_service.stdout }}"
+
+        - name: Replace the placeholder for service endpoint
```

Comment:
@sushma1118 This same name is used for previous task also can you change it.

Reply:
ok

Reply:
Changed the task name